libraries/AP_AdvancedFailsafe/AP_AdvancedFailsafe.cpp

```diff
@@ -145,6 +145,13 @@ const AP_Param::GroupInfo AP_AdvancedFailsafe::var_info[] = {
     // @Units: s
     AP_GROUPINFO("RC_FAIL_TIME",   19, AP_AdvancedFailsafe, _rc_fail_time_seconds,    0),
 
+    // @Param: TERM_DLY
+    // @DisplayName: Termination Delay
+    // @Description: This is the time in seconds the aircraft will stay at a failsafe waypoint before aero termination.  Set to 0 to disable.
```

Comment:
I'm not very familiar with AFS code, but from a quick look this seems to do something new (and the description isn't really explicit on this).

From what I can tell, previously GCS or GPS individual failure would never trigger a terminate action (there is a parameter to cause termination if both failures happen). That's what the new code adds with a single parameter.

I would not only recommend two separate parameters, but to make the description (and Display name) more accurate - this isn't a delay to termination, this enables termination on two previously not-terminated causes.